modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc

```diff
@@ -382,37 +382,63 @@ To enable the SSL checks, the certificate must be imported following these steps
-
-Last but not least, and extending the mechanisms described above to provide the authentication token, there is another mechanism that allows you to implement a custom token provider to allow the connector requests for security tokens when needed. It is your responsibility to solve each request and to return a fresh token every time the connector asks for one. As the token has an expiration time within it, this mechanism allows the connector to re-authenticate with the target resource once the token has expired (this is not allowed with the previous mechanisms described above where the tokens are fixed at configuration time before the flow runs).
+== SAS-Based Authentication
 
-The following are the available settings for the `Shared Access Signature` connection:
+In addition to the connection schemes that require a username and password, the connector provides a connection in which authentication is SAS-based (only for Azure). This type of connection enables you to set the authentication token for the Service Bus service without requiring a username and password.
 
-Service Namespace: The namespace for the Azure Service Bus Service within your subscription.
+Because the SAS token schema is URI based (that is, you can assign different authorization access to your resources based on their URIs), the connection supports multiple ways of providing the needed authentication tokens. The simplest case is when you have one profile that authorizes accesses to all of your resources using a root URI that is the base endpoint that your service exposes. To provide different access tokens on different resources, use a setting that enables you to configure a list of tokens according to the resources the connector needs to access when it runs.
 
-Shared Access Signature: (Optional) The unique security token when you set up just one security profile to access all your service resources. In this case after the token expires, the connector cannot reconnect automatically and you need to stop running the flow to update this configuration setting with a new token.
+As an extension to the mechanisms described previously, you can also implement a custom token provider that enables the connector to request security tokens when it needs them. It is the responsibility of the app to solve each request and to return a fresh token every time the connector asks for one. Because the token contains an expiration time, this mechanism allows the connector to re-authenticate with the target resource once the token expires. Re-authentication is not allowed when the tokens are specified at configuration time.
```

Comment:
Per the above
```suggestion
You can also implement a custom token provider that enables the connector to request security tokens from the app as it needs them, so that the tokens are always "fresh." Because the tokens expire, this mechanism allows the connector to reauthenticate with the target resource once the token expires, which is not allowed when the tokens are specified at configuration.
```